profiles/models.py

```diff
@@ -1,14 +1,24 @@
 from django.db import models
 from django.contrib.auth.models import User
 
-class Milestone(models.Model):
-    good_start = models.BooleanField(default=False)
-    terrific_start = models.BooleanField(default=False)
-    almost_there = models.BooleanField(default=False)
-    mission_accomplished = models.BooleanField(default=False)
-
-
 class Leaderboard(models.Model):
     username = models.ForeignKey(User, on_delete=models.CASCADE)
     points = models.IntegerField()
-    milestones = models.ForeignKey(Milestone, on_delete=models.CASCADE)
+    good_first_issue = models.BooleanField(default=False) #Shall be true on completion of Good first issue
+    medium_question_1 = models.BooleanField(default=False) #Shall be true on completion of first medium issue
+    medium_question_2 = models.BooleanField(default=False) #Shall be true on completion of first medium issue
+    '''
+    We can identify labels for various situations as follows
```

Comment:
labels here means?? Sorry if I sound dumb... :joy:

Reply:
No you don't sound dumb, and by labels I meant `good_start`, `terrific_start`, `almost_there` and `mission_accomplished` 😅

Reply:
in the views or something... when we send it to the frontend??

Reply:
Yeah